### DISCUSSION TITLE
Section 03 DataLoader

### DISCUSSION THREAD

**Discussion by mavaylon1** [mrdbourke/pytorch-deep-learning#601] [Q&A]
In Section 02, we convert the labels to LongTensors due to CrossEntropyLoss. But in Section03, we do not and it still works. Why?
